content/departments/product-engineering/engineering/code-graph/code-insights/processes.md

```diff
@@ -46,6 +46,24 @@ Our current process is as follows:
 
 - If one or more issues that were planned for an iteration are looking to **not get finished** (which includes testing) in the [current iteration](https://github.com/orgs/sourcegraph/projects/200/views/1) (while maintaining sustainable work practices) the assignee **raises this as soon as possible asynchronously** to the team (including the PM and EM), e.g. on the GitHub issue or Slack. These issues then **become _proposed_ issues for the next iteration** (meaning nothing carries over automatically, but we also don't just drop and forget missed issues).
 
+## How we think about issue sizes
+
+When sizing an issue please keep in mind the size includes the entire lifecycle of the issue. The coding, any testing both automated and manual, as well as PR reviews, documentation, and any follow-up needed.
```

Comment:
> and any follow-up needed

This sticks out to me as being fairly well outside the bounds of a single issue. If there is truly some follow up tasks, should that not be separate issues? If there is unknown follow up, how would we plan to include that?

Reply:
Excellent point! Re-wording.